share/spice/sound_cloud/sound_cloud.js

```diff
@@ -2,75 +2,75 @@
     "use strict"
 
     env.ddg_spice_sound_cloud = function(api_result) {
-
-        var SOUNDCLOUD_CLIENT_ID = 'df14a65559c0e555d9f9fd950c2d5b17',
-            script = $('[src*="/js/spice/sound_cloud/"]')[0],
+        var script = $('[src*="/js/spice/sound_cloud/"]')[0],
             source = $(script).attr("src"),
-            query = source.match(/sound_cloud\/([^\/]*)/)[1],
-
+            query = source.match(/sound_cloud\/([^\/]*)/)[1];
+        
+        $.getJSON("/js/spice/sound_cloud_result/" + query, function(api_result) {
```

Comment:
don't name it `api_result`, that's confusing with the above. change one or the other